components/binance/browser/binance_service_browsertest.cc

```diff
@@ -221,6 +222,12 @@ class BinanceAPIBrowserTest : public InProcessBrowserTest {
     base::PathService::Get(brave::DIR_TEST_DATA, &test_data_dir);
   }
 
+  void SetUpCommandLine(base::CommandLine* command_line) override {
+    // HTTPS server only serves a valid cert for localhost, so this is needed
+    // to load pages from other hosts without an error.
+    command_line->AppendSwitch(switches::kIgnoreCertificateErrors);
```

Comment:
https://bravesoftware.slack.com/archives/C7VLGSR55/p1636702893335800

pls replace with `content::ContentMockCertVerifier` instead.

Reply:
Fixed in 319a1793b707cd9bf12907e1e8702d2d11f627fc.

Reply:
Further fix in 6a9bd9b2ce2c8a8195d618f1075725fdb371ff57